migrator/migrations/m_181_to_m_182_remove_default_scope_manager_role/migration_impl.go

```diff
@@ -0,0 +1,205 @@
+package m181tom182
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	apiTokenStore "github.com/stackrox/rox/migrator/migrations/m_181_to_m_182_remove_default_scope_manager_role/apitokenstore"
+	groupStore "github.com/stackrox/rox/migrator/migrations/m_181_to_m_182_remove_default_scope_manager_role/groupstore"
+	permissionSetStore "github.com/stackrox/rox/migrator/migrations/m_181_to_m_182_remove_default_scope_manager_role/permissionsetstore"
+	roleStore "github.com/stackrox/rox/migrator/migrations/m_181_to_m_182_remove_default_scope_manager_role/rolestore"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+const (
+	scopeManagerObjectName = "Scope Manager"
+
+	oldScopeManagerDescription = "For users: use it to create and modify scopes for the purpose of access control or vulnerability reporting"
+
+	updatedDescriptionSuffix = ". This used to be a system object and should have been removed but was still " +
```

Comment:
Should we also create a prefix of `DEPRECATED` like you originally had before for the name? I think it'd be worthwhile to catch people's attention when they are browsing the role view with the prefix, otherwise they may not notice it from the beginning.